src/i18n/strings/en_EN.json

```diff
@@ -15,6 +15,43 @@
     "Unknown device": "Unknown device",
     "%(appName)s (%(browserName)s, %(osName)s)": "%(appName)s (%(browserName)s, %(osName)s)",
     "You need to be using HTTPS to place a screen-sharing call.": "You need to be using HTTPS to place a screen-sharing call.",
+    "Downloading": "Downloading",
+    "Decrypting": "Decrypting",
+    "Scanning...": "Scanning...",
+    "Download": "Download",
+    "Infected content": "Infected content",
+    "Attachment": "Attachment",
+    "Image": "Image",
+    "Sticker": "Sticker",
+    "Video": "Video",
+    "Error processing audio message": "Error processing audio message",
+    "Download %(text)s": "Download %(text)s",
+    "Error": "Error",
+    "Error decrypting attachment": "Error decrypting attachment",
+    "Decrypt %(text)s": "Decrypt %(text)s",
+    "Invalid file%(extra)s": "Invalid file%(extra)s",
+    "Error decrypting image": "Error decrypting image",
+    "Show image": "Show image",
+    "Error decrypting video": "Error decrypting video",
+    "Error processing voice message": "Error processing voice message",
+    "Scanning": "Scanning",
+    "Scan done": "Scan done",
```

Comment:
The text we have on the android client is "Trusted", let's reuse the same.
We also need a french translation, "Vérifié"

Reply:
It is now „Trusted“/„Vérifié“ . Moved the translations to `tchap_translations.json`.